### ISSUE TITLE
LaTeX blocks with long lines become decentered and overflow the node bounding box (for some rendering environments)

### ISSUE THREAD

**Issue by smheidrich** [terrastruct/d2#2083]
Consider:

```d2
a: {
  eqn: |latex
    a\\
    super\\ super\\ super\\ super\\ super\\ super\\
    super\\ super\\ super\\ super\\ super\\ super\\
    long\\ equation
  |
}
```

This will sometimes render OK and sometimes cause the long equation line to be shifted to one side, depending on the rendering environment:

<details open>
  <summary>Rendered to an SVG using Dagre on my system</summary>
  <img src="https://github.com/user-attachments/assets/be0c2fa8-d7db-4e19-822a-e39d69193adc">
  <br/>
</details>

<details open>
  <summary>Rendered to an SVG using ELK on my system</summary>
  <img src="https://github.com/user-attachments/assets/47dcb149-e5f2-4664-b8d5-e7013b3c0fda">
  <br/>
</details>

<details>
  <summary>Rendered using Dagre in the D2 Playground and exported (same as on my system)</summary>
  <img src="https://github.com/user-attachments/assets/3be6638b-ceb9-4214-851f-2b0ade165b1f">
</details>

Curiously, it gets shifted in the other direction in the preview panel of the D2 Playground (and the shift is less pronounced so it stays within its bounding box at this length):

<details open>
  <summary>Screenshot of the D2 Playground</summary>
  <img src="https://github.com/user-attachments/assets/1a472c10-118a-4b6e-8486-b1f89748368c">
</details>

Nothing of the sort happens when I write similarly long labels for regular non-LaTeX nodes, so this seems to be LaTeX-specific.

### System information

Dagre version: 0.6.6 (on my system, no idea what the Playground uses)

**Comment by alixander** [comment]
can you give more system information? e.g. browser, os

this renders and exports without overflow for me: https://play.d2lang.com/?script=SrRSqOZSUEgtzLNSqMlJLEmt4FJQUFBIjIkB08WlBalFMTHE0mTrycnPS4-JUUgtLE0syczP41JQqOGq5QIEAAD__w%3D%3D&

-----------

it does look like it consistently underflows though, e.g. the box is longer than contents

**Comment by smheidrich** [comment]
@alixander Sure (I didn't realize the Playground export was entirely client-side): Debian 12, Firefox 130.0

But I just tried it on my work machine which runs Ubuntu 22.04 with the same Firefox version and it looked _perfect_ there.

So my guess is this is something specific to my exact system and if I did a fresh install of Debian it would probably be fine.

I'll close this, thanks for looking into it!
